drizzlepac/hlautils/se_source_generation.py

```diff
@@ -315,14 +330,14 @@ def compute_background(image, box_size=50, win_size=3, nsigma=5., threshold=None
             default_threshold = bkg.background + bkg_rms
             bkg_rms_mean = bkg.background.mean() + nsigma * bkg_rms.std()
             bkg_dao_rms = bkg.background_rms
-            if threshold is None:
+            if threshold_flag is None:
                 threshold = default_threshold
-            elif threshold < 0:
-                threshold = -1 * threshold * default_threshold
+            elif threshold_flag < 0:
+                threshold = -1 * threshold_flag * default_threshold
                 log.info("{} based on {}".format(threshold.max(), default_threshold.max()))
```

Comment:
This print statement should have been revised weeks ago in the original code to say something like "background threshold set to {} based on {}".